README.md

```diff
@@ -54,3 +54,22 @@ $ WS_SECRET="secret2" npm start
 - frontend.secret: Secret to be used in the ethstats proxy.
 
 - save-block-txs: Whether block transactions should be written to database.
+
+
```

Comment:
Can we also raise a PR against `matic-cli` ?
That would allow us to have a dedicated built-in command which clones the repo, builds the docker services, and runs the containers on machine0.
Thanks

Reply:
Yes, that is already in progress. These changes are to just make it compatible with express-cli remote command.